experiments/list_unsupported.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/sh
 
+# Usage info
+usage()
+{
+   echo "Run GNAT2Goto on an Ada repository.\n"
+   echo "The output is an ordered list of currently unsupported features"
+   echo "with the number of times they occur in the input repository.\n"
+   echo "The script builds a parsing program using collect_unsupported.cpp and expects"
+   echo "this file to be in the same folder.\n"
+   echo "Usage:\n\nlist_unsupported.sh path_to_ada_source_folder"
```

Comment:
I think this should probably be the first line?

Reply:
Ah, good point - yes, I'll change that.